test-cases/test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
this is ok for now, another more harmonious solution is to use test containers, spin up minikube, launch wf's, party